node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -2104,10 +2104,7 @@ mod tests {
 
         let result_introducer: &NodeRecord =
             dest_db.node_by_key(&agrs[0].inner.public_key).unwrap();
-        let mut expected_introducer = NodeRecord::from(&agrs[0]);
-        expected_introducer.metadata.last_update = result_introducer.metadata.last_update;
-        expected_introducer.metadata.country_undesirability = COUNTRY_UNDESIRABILITY_FACTOR;
-        expected_introducer.resign();
+        let expected_introducer = NodeRecord::from(&agrs[0]);
```

Comment:
I'm curious about why the `last_update` and `country_undesirability` changes to `expected_introducer` are no longer required to make it line up with `result_introducer`.